.pre-commit-config.yaml

```diff
@@ -66,13 +66,13 @@ repos:
         args: ['--count', '--select', 'E101,E11,E111,E112,E113,E121,E122,E123,E124,E125,E126,E127,E128,E129,E131,E133,E20,E211,E231,E241,E242,E251,E252,E26,E265,E266,E27,E301,E302,E303,E304,E305,E306,E401,E402,E502,E701,E711,E712,E713,E714,E722,E731,E901,E902,F822,F823,W191,W291,W292,W293,W391,W601,W602,W603,W604,W605,W690']
```

Comment:
Why are these changes in the diff?

Reply:
I messed up in git, let me check.

Reply:
If you are having issues, it might be better to checkout the old version of this PR. Squash each commit and then rebase it.

Reply:
Resolved now.